src/libraries/System.Drawing.Common/src/System.Drawing.Common.csproj

```diff
@@ -5,14 +5,14 @@
     <WarningsNotAsErrors>CS0618</WarningsNotAsErrors>
     <DefineConstants Condition="'$(TargetsWindows)' == 'true'">$(DefineConstants);FEATURE_WINDOWS_SYSTEM_COLORS;FEATURE_SYSTEM_EVENTS</DefineConstants>
     <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
-    <TargetFrameworks>$(NetCoreAppCurrent)-Windows_NT;$(NetCoreAppCurrent)-Unix;$(NetCoreAppCurrent)-Browser;netcoreapp3.0-Windows_NT;netcoreapp3.0-Unix</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent)-Windows_NT;$(NetCoreAppCurrent)-Unix;$(NetCoreAppCurrent);netcoreapp3.0-Windows_NT;netcoreapp3.0-Unix;netcoreapp3.0</TargetFrameworks>
```

Comment:
Why are we adding `netcoreappp3.0` here?

Reply:
We want anybody using system.Drawing.common package on browser to have pnse, which is not the case currently. net5.0 is not included in the package, only netcoreapp3.0-win & netcoreapp3.0-unix.

Adding a version less config netcoreapp3.0 will have pnse for browser or any other os not a child of windows & unix.

Reply:
> net5.0 is not included in the package

That's the part I was missing. Thanks.

Reply:
Before this would have resolved to the `netstandard2.0` lib which is harvested, but presumably that was a ref-def mismatch because it was lower version than the 3.0 ref?

Reply:
yes thats a correct. The 3.0 contains some api changes which are not in netstandard2.0